tools/awps-tunnel/client/package.json

```diff
@@ -76,7 +77,8 @@
   "scripts": {
     "clean": "rimraf ./build",
     "start": "cross-env REACT_APP_DATA_FETCHER=mock react-scripts start",
-    "build": "npm run build:mock",
+    "prebuild": "node scripts/downloadExamples.js",
```

Comment:
naming: `downloadSpec.js`? and have the version path as an input parameter here, e.g. `node scripts/downloadSpec.js 2024-01-01`